3_acm/cloudbuild.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s # 1hr
+tags:
+  - modern-cicd-anthos
+  - modern-cicd-anthos-acm
+substitutions:
+  _DOMAIN: ""
+  _BUILD_IMAGES: "true"
+options:
+  substitution_option: 'ALLOW_LOOSE'
+steps:
+- name: 'gcr.io/${PROJECT_ID}/anthos-platform-installer'
```

Comment:
Should we switch to using https://registry.terraform.io/modules/terraform-google-modules/kubernetes-engine/google/6.0.1/submodules/acm?

Reply:
@henrybell can you weigh in on this since I think you're setting up Endpoints?

@morgante -- We need to think about how to get the ACM repo out of GitLab after it had just been created.  In theory, the GitLab instance is already live, so it should be possible.

I would suggest now making the change in this PR, but file an enhancement Issue to add the change

Reply:
Issue #48 -- Added so we can track the change after the PR goes through.